doc/user_guide/user-guide.md

```diff
@@ -38,12 +38,12 @@ You can also provide additional configurations:
 Before starting the application, you need to set an environment variable that points to the properties file:
```

Comment:
Can you add a link to the generated docs?
Since they are not there yet maybe a ticket for that?